test/benchmark/nginx/http_req10k_conc1_bw/config.json

```diff
@@ -0,0 +1,9 @@
+{
+    "alert_threshold": "125%",
+    "alert_tool": "customBiggerIsBetter",
+    "search_pattern": "Transfer rate:",
+    "result_index": "3",
+    "description": "ab -n 10000 -c 1 http://localhost:8080/index.html",
+    "title": "[Network] Nginx HTTP request performance",
```

Comment:
Remove `[Network]`. Every benchmark results on Nginx is about _network_. So unlike LMbench, such a prefix prvoides no useful information at all.

Reply:
In addition to remove `[Network]`, I think it should also adding some information about the `-n 10000 -c 1`. So I change it to `Nginx HTTP request performance with 1 concurrency and 10000 requests in total`